index.php

```diff
@@ -36,48 +42,81 @@ function displayAllRecords(array $records): string
     return $htmlOutput;
 }
 
-// Handle add record
+//display genres function
+function displayAllGenres(array $genres):string
```

Comment:
You should have a space between : and string

Reply:
Also, at this point I think it would be worth moving this function and displayAllRecords into their own files (don't forget to update the require statements in your tests)